src/main/java/io/aiven/kafka/connect/opensearch/OpensearchSinkConnectorConfig.java

```diff
@@ -157,13 +157,16 @@ public class OpensearchSinkConnectorConfig extends AbstractConfig {
     private static final String BEHAVIOR_ON_MALFORMED_DOCS_DOC = "How to handle records that "
             + "OpenSearch rejects due to some malformation of the document itself, such as an index"
             + " mapping conflict or a field name containing illegal characters. Valid options are "
-            + "``ignore``, ``warn``, and ``fail``.";
+            + "``ignore`` - do not index the record, ``warn`` - log a warning message and do not index the record, "
```

Comment:
@C0urante could you please update docs [1] with these configuration changes? cc @willyborankin 

[1] https://github.com/aiven/opensearch-connector-for-apache-kafka/blob/main/docs/opensearch-sink-connector-config-options.rst